### PR TITLE
Feature/add button

### PR DESCRIPTION
**Pull request by markjehn**
I just created button in index.html, and I also created css file for button. 
Screenshot 2024-09-18 at 8 37 27 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,11 +4,33 @@
...
+        <textarea class="modal_task_textarea" id="task_description_input" placeholder="Give a task description here..." rows="10" cols="30"></textarea>
+      </div>
+    </div>
+    <div class="button_container">
+      <button class="cancel_button" id="cancel_modal_button">Cancel</button>
+      <button class="apply_button" id="apply_modal_button">Apply</button>
+    </div>
+  </div>
+  <!-- Add button -->
+  <button class="add-button">+</button>
```
Comment (savchyk-vlad) [inline index.html]: Can you move your `<button class="add-button">+</button>` above the modal window by structure? Modal window html should be at the end of html body but before script link. Our button should be under the gray placeholder.

like this
Screenshot 2024-09-18 at 11 47 27 PM

you don't need create new branch
just make changes in same branch -> commit your changes -> and push your changes again

Comment (markjehn) [reply index.html]: got it. thanks vlad.

**Review by savchyk-vlad — APPROVED** [review]
Cool 👏

**Review by celadon29 — APPROVED** [review]
nice button

**Pull request merged**